holidays/countries/india.py

```
@@ -492,12 +515,121 @@ def _populate_subdiv_br_public_holidays(self):
+        name = tr("Jor Mela Fatehgarh Sahib")
+        self._add_holiday_dec_26(name)
+        self._add_holiday_dec_27(name)
+        self._add_holiday_dec_28(name)
 
     # Chhattisgarh.
     def _populate_subdiv_cg_public_holidays(self):
-        # Chhattisgarh Foundation Day.
-        self._add_holiday_nov_1(tr("Chhattisgarh Foundation Day"))
+        pass
+
+    def _populate_subdiv_cg_government_holidays(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_populate_subdiv_cg_government_holidays`

Add return type annotation: `None`

(ANN202)